packages/uniforms/__tests__/useField.tsx

```diff
@@ -1,100 +1,99 @@
+import { fireEvent, screen } from '@testing-library/react';
 import React, { ReactNode } from 'react';
 import { AutoForm, BaseForm, connectField, useField } from 'uniforms';
 import { JSONSchemaBridge } from 'uniforms-bridge-json-schema';
+import { SimpleSchema2Bridge } from 'uniforms-bridge-simple-schema-2';
 
-import mount from './_mount';
+import { render } from '../__suites__';
 
-describe('useField', () => {
-  const bridge = new JSONSchemaBridge(
-    {
-      type: 'object',
-      properties: {
-        a: { type: 'string' },
-        b: { type: 'object', properties: { c: { type: 'string' } } },
-        d: { type: 'number', default: 4 },
-      },
+const bridge = new JSONSchemaBridge(
+  {
+    type: 'object',
+    properties: {
+      a: { type: 'string' },
+      b: { type: 'object', properties: { c: { type: 'string' } } },
+      d: { type: 'number', default: 4 },
     },
-    () => {},
-  );
+  },
+  () => {},
+);
+
+function Test(rawProps: {
+  children?: ReactNode;
+  name: string;
+  options?: Parameters<typeof useField>[2];
+}) {
+  const [props] = useField(rawProps.name, rawProps, rawProps.options);
+  return <>{props.children}</>;
+}
 
-  function Test(rawProps: {
-    children?: ReactNode;
-    name: string;
-    options?: Parameters<typeof useField>[2];
-  }) {
-    const [props] = useField(rawProps.name, rawProps, rawProps.options);
-    return <>{props.children}</>;
-  }
+const TestComponent = connectField((props: Record<string, any>) => {
+  return (
+    <input
+      value={props.value || ''}
+      onChange={event => {
+        props.onChange(event.target.value);
+      }}
+    />
+  );
+});
 
+describe('useField', () => {
   it('is a function', () => {
     expect(useField).toBeInstanceOf(Function);
   });
 
   describe('when called with initialValue', () => {
-    const TestComponent = connectField((props: Record<string, any>) => {
-      return (
-        <input
-          value={props.value || ''}
-          onChange={event => {
-            props.onChange(event.target.value);
-          }}
-        />
-      );
-    });
-
     it('applies default value', () => {
-      const wrapper = mount(
+      render(
         <AutoForm schema={bridge}>
           <TestComponent name="d" />
         </AutoForm>,
+        { schema: { type: SimpleSchema2Bridge } },
       );
+      const input = screen.getByRole('textbox');
 
-      expect(wrapper.find('input').prop('value')).toBe(4);
-
-      expect(
-        wrapper
-          .find('input')
-          .simulate('change', { target: { value: undefined } }),
-      ).toBeTruthy();
+      expect(input).toHaveAttribute('value', '4');
     });
 
     it('does not apply default value after first change', () => {
-      const wrapper = mount(
-        <AutoForm schema={bridge}>
+      const { getByRole } = render(
```

Comment:
I would always call it the same. Either `screen` or `renderResult` (then the same in `render.tsx`).